api/src/main/java/org/openmrs/module/ohricore/api/impl/HIVStatusComputedConcept.java

```diff
@@ -55,15 +34,17 @@ Is Positive (condition 2):
 
         for (Obs obs : hivTestObs) {
 
+            if (obs.getVoided()) {
+                continue;
+            }
+
             Concept obsValueCoded = obs.getValueCoded();
             if (obsValueCoded.getUuid().equals(CommonsUUID.POSITIVE)) {
```

Comment:
Could we have the concept here instead of comparing with the uuid? 
if obsvaluecoded == POSITIVE_CONCEPT (not uuid)? same for NEGATIVE?

Reply:
@alaboso doesn't this mean some 2 extra trips to the database to fetch the concepts by UUID to be used for comparison? If it is not an issue, let me go ahead and make that change.